website/cue/reference/components/sources/internal_logs.cue

```diff
@@ -152,4 +152,11 @@ components: sources: internal_logs: {
 				"""
 		}
 	}
+
+	telemetry: metrics: {
+		component_discarded_events_total:     components.sources.internal_metrics.output.metrics.component_discarded_events_total
+		component_errors_total:               components.sources.internal_metrics.output.metrics.component_errors_total
```

Comment:
Are we emitting these two? I don't see them.

Reply:
yes, with `StreamClosedError`